src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -385,7 +414,7 @@ private class ClientInfo
             public string Description { get; }
             public INamedTypeSymbol? ExistingType { get; }
             public IReadOnlyList<InputOperation> Operations { get; }
-            public IReadOnlyList<InputParameter> ClientParameters { get; }
+            public IReadOnlyList<InputParameter> ClientParameters { get; set; }
```

Comment:
Instead of making this settable, I would rather prefer we calculate this when we call it something like
```suggestion
            public IReadOnlyList<InputParameter> ClientParameters => _clientParameters ??= Children.SelectMany(...);
```

Reply:
updated